lib/runtime-c-api/src/lib.rs

```diff
@@ -1436,14 +1421,12 @@ impl From<Type> for wasmer_value_tag {
 
 impl From<wasmer_value_tag> for Type {
     fn from(v: wasmer_value_tag) -> Self {
-        unsafe {
```

Comment:
I don't understand why the `match` statement is in an `unsafe` block. Did I miss something?

Reply:
The C API is fairly new and many things like this needed clean up.